en/sensor/optical_flow.md

```diff
@@ -34,11 +34,19 @@ The output of the flow when moving in different directions must be as follows:
 | Right            | - X             |
 | Left             | + X             |
 
-For pure rotations the `integrated_xgyro` and `integrated_x` (respectively `integrated_ygyro` and `integrated_y`) have to be the same.
-
 Sensor data from the optical flow device is fused with other velocity data sources.
 The approach used for fusing sensor data and any offsets from the center of the vehicle must be configured in the [estimator](#estimators).
 
+### Scale Factor
+
+For pure rotations the `integrated_xgyro` and `integrated_x` (respectively `integrated_ygyro` and `integrated_y`) have to be the same.
```

Comment:
Thanks @bresch !

I'm going to merge this, but can you tell me what these areL

>  `integrated_xgyro` and `integrated_x` (respectively `integrated_ygyro` and `integrated_y`)

Specifically they don't appear anywhere else in the doc so there is no way for someone to know if they are the same or otherwise. Ideally having a link to more info would be good.

Reply:
Those values are in the [mavlink message](https://mavlink.io/en/messages/common.html#OPTICAL_FLOW_RAD), so the user can easily monitor that in the mavlink inspector

Reply:
Thanks. I spelled it out in #3526